specification/eventhub/resource-manager/Microsoft.EventHub/stable/2024-01-01/examples/NameSpaces/EHNamespaceFailover.json

```
@@ -0,0 +1,22 @@
+{
+  "parameters": {
+    "namespaceName": "NamespaceGeoDRFailoverSample",
+    "resourceGroupName": "ResourceGroupSample",
+    "api-version": "2024-01-01",
+    "subscriptionId": "SampleSubscription",
+    "parameters": {
+      "properties": {
+        "primaryLocation": "centralus",
+        "maximumGracePeriodInMins": 5
+      }
+    }
+  },
+  "responses": {
+    "202": {
+      "headers": {
+        "azure-asyncoperation": "http://azure.async.operation/status",
+        "location": "eastus"
```

This line is the cause of the runtime failure in the BreakingChange PR check. Here is the fix:
```suggestion
        "location": "http://azure.async.operation/status"
```

---

Thanks Mike!